__tests__/app.test.js

```diff
@@ -96,3 +98,48 @@ describe("get /api/topics", () => {
       });
   });
 });
+
+describe("test4.get /api/articles/:article_id", () => {
+  test("200: responds with an object of article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        // console.log(response);
+
+        //   expect(message).toBeInstanceOf(Object);
+        // expect(testStack.hasOwnProperty("quantity")).toBe(true);
+
+        expect(typeof response).toBe("object");
+        expect.objectContaining({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+        });
+      });
+  });
+});
+
+describe("test4.get /api/articles/:article_id", () => {

```

Comment:
if this is testing the same end point it should ideally be in the same describe block to make for drier code.